internal/boxcli/init.go

```diff
@@ -28,7 +30,15 @@ func initCmd() *cobra.Command {
 			"You can then add packages using `devbox add`",
 		Args: cobra.MaximumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runInitCmd(cmd, args, flags)
+			err := runInitCmd(cmd, args, flags)
+			if errors.Is(err, os.ErrExist) {
+				path := pathArg(args)
+				if path == "" || path == "." {
+					path, _ = os.Getwd()
+				}
+				return usererr.New("devbox.json already exists in %q.", path)
```

Comment:
Could we make switch this to a warning emitted instead and make it a non-error? 
I don't think its _that_ bad to run `devbox init` if a devbox.json already exists. Its useful for scripts.